physics/malus_law.py

```diff
@@ -0,0 +1,82 @@
+import doctest
```

Comment:
This is only used in the
`if __name__ == "__main__`
and therefore should be imported there
```suggestion
```

Reply:
Ok Thanks